Proton/Sources/Editor/EditorView.swift

```diff
@@ -1020,7 +1027,7 @@ extension EditorView {
 
             if attachment.isRendered == false {
                 attachment.render(in: self)
-                if let focusable = attachment.contentView as? Focusable {
+                if !isSettingAttributedText, let focusable = attachment.contentView as? Focusable {
```

Comment:
My test cases:

1. Loading MiniEditor with Draft which contains Panel (used to focus, doesn’t any more)
2. Loading FullEditor with a Description which contains a Panel (used to focus, doesn’t any more)
3. Inserting a panel into an Editor (should focus, and still does)